apis/apim/v1alpha1/ratelimitpolicy_types.go

```diff
@@ -53,8 +57,8 @@ type MetadataSpec struct {
 	MetadataKey   MetadataKeySpec `json:"metadata_key"`
 	// +optional
 	DefaultValue *string `json:"default_value,omitempty"`
-	// +optional
```

Comment:
It is still optional, right? I think that kubebuilder does not do anything with `// +optional` comment, but I would keep it as for the reader it makes it very clear.

The `// +kubebuilder:default=DYNAMIC` comment only affects deserialization, right? on serialization, if it is empty string `""`) it will just not be added in the json strinc.

Reply:
I think default itself means optional because if it's not provided in the manifest, it will be defaulted to `DYNAMIC`.